.NET/Microsoft.Recognizers.Text.Number/Models/AbstractNumberModel.cs

```diff
@@ -88,11 +88,6 @@ private ModelResult BuildModelResult(ParseResult pn)
                     if (pn.Metadata != null && pn.Metadata.IsOrdinalRelative)
                     {
                         specificNumberType = Constants.MODEL_ORDINAL_RELATIVE;

```

Comment:
Why can't we also do this somewhere in the parser instead of having strange logic at the model level?
It seems most everything else was already moved.